lib/app/features/users/data/repositories/users_repository.dart

```diff
@@ -82,4 +113,12 @@ extension _FutureExtension<T extends String> on Future<T> {
     return then((v) => jsonDecode(v) as Map<String, dynamic>)
         .then((v) => UserSearchSessionModel.fromJson(v));
   }
+
+  Future<ValidField> parseBlockOrReport() async {
+    return then((data) async {
+      final jsonData = jsonDecode(data) as Map<String, dynamic>;
+      String message = jsonData['message'];
+      return ValidField(message: message);
```

Comment:
aqui dá pra usar a factory `fromJson`:
```suggestion
      return ValidField.fromJson(jsonData);
```